src/test/scala/mesosphere/marathon/integration/AppDeployIntegrationTest.scala

```diff
@@ -44,23 +44,24 @@ class AppDeployIntegrationTest
     deploymentIds.length should be(1)
     val deploymentId = deploymentIds.head
 
-    val apiPostEvent = waitForEvent("api_post_event")
+    log.info("waiting for deployment success")
+    val events: Map[String, Seq[CallbackEvent]] = waitForEvents(
```

Comment:
Why do you list events here, that never get accessed in the test afterwards? (e.g. deployment_info)?